### PR TITLE
Added custom config for incremetal rings

### PR DESCRIPTION
**Pull request by Akreddy123**

### CODE REVIEW

File: src/service/Domain/FeatureFlag.cs [hunk 1]
```diff
@@ -13,6 +13,8 @@ public class FeatureFlag
         public string Description { get; set; }
         [JsonProperty("enabled")]
         public bool Enabled { get; set; }
+        [JsonProperty("enableIncrementalRings")]
+        public bool EnableIncrementalRings { get; set; }
```
Comment (patrickCode) [inline src/service/Domain/FeatureFlag.cs:17]: To maintain consistency rename to `IncrementalRingsEnabled`

**Comment by patrickCode** [comment]
Make necessary changes in `ActivateStage` API in `FeatureFlagsController`.

File: src/service/Domain/FeatureFlag.cs (hunk 1 above)
Comment (Akreddy123) [reply src/service/Domain/FeatureFlag.cs]: done

**Comment by Akreddy123** [comment]
> Make necessary changes in `ActivateStage` API in `FeatureFlagsController`.

done

**Review by patrickCode — APPROVED** [review]
LGTM

**Review by deepakparamesh — APPROVED** [review]

**Review by patrickCode — APPROVED** [review]

**Review by pratikgolchha — APPROVED** [review]

**Pull request merged**